src/controller/user.js

```diff
@@ -0,0 +1,42 @@
+const { getAllUsersService, createUserService } = require('../service/user')
+const { checkExistingUserOrEmail } = require('../dao/user')
+const bcrypt = require('bcrypt');
+
+const getAllUsers = async (req, res) => {
+  try {
+    const users = await getAllUsersService()
+    res.status(200).json(users)
+  } catch (error) {
+    res.status(500).json(error)
+  }
+}
+
+
+const createUser = async (req, res) => {
+  try {
+    const user = req.body.user
+    const FirstPassword = req.body.password
+    const email = req.body.email
```

Comment:
you can use destructuring to get in an easy way the variables needed.
with : you can assign an alias like initialPassword
` const { user, passwordi: nitialPassword, email } = req.body;`